src/main/resources/templates/datewarehouse/etl-jobs-config.json

```diff
@@ -94,30 +276,107 @@
 		"sourcePath" : "log.sagebase.org/fileSnapshots/",
 		"tableDescriptor": {
 			"name": "filesnapshots",
-			"columns": {
-				"change_type": "string",
-				"change_timestamp": "timestamp",
-				"change_user_id": "bigint",
-				"snapshot_timestamp": "timestamp",
-				"id": "bigint",
-				"created_by": "bigint",
-				"created_on": "timestamp",
-				"modified_on": "timestamp",
-				"concrete_type": "string",
-				"content_md5": "string",
-				"content_type": "string",
-				"file_name": "string",
-				"storage_location_id": "bigint",
-				"content_size": "bigint",
-				"bucket": "string",
-				"key": "string",
-				"preview_id": "bigint",
-				"is_preview": "boolean",
-				"status": "string"
-			},
-			"partitionKeys": {
-				"snapshot_date": "date"
+			"description" : "This table contain snapshots of file handles. Snapshots are taken when file handles are created, updated or deleted. Note: Snapshots are also taken periodically and independently of the changes. The snapshot_timestamp records when the snapshot was taken.",
```

Comment:
Same comment as above:

is there a column that indicates this: "Snapshots are also taken periodically and independently of the changes" so we can query these snapshots vs nodes that are created, updated, or deleted

Reply:
Same explanation as above